xtable-core/src/test/java/org/apache/xtable/iceberg/TestIcebergSync.java

```diff
@@ -170,9 +170,9 @@ public class TestIcebergSync {
           .build();
   private final Schema icebergSchema =
       new Schema(
-          Types.NestedField.required(1, "timestamp_field", Types.TimestampType.withoutZone()),
```

Comment:
Why does this numbering change? Would it break when this version gets rolled out to existing iceberg targets?

Reply:
This is just to prove out that the field numbering matches what is provided. Currently the numbers are reassigned when the table is first created.

This can impact existing tables if the schema field IDs are currently wrong but I think that would also imply that existing table is in a broken state.